### PR TITLE
 feature/CD-93

### PR DESCRIPTION
**Pull request by Gerasimov94**

### CODE REVIEW

**Comment by gusartus** [comment]
ImageOfError
🤔

File: src/modules/RegAuth/Registration/reducer/registrationReducer.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {handleActions} from 'redux-actions';
```
Comment (CodeAnnihilator) [inline src/modules/RegAuth/Registration/reducer/registrationReducer.ts:1]: image

Comment (CodeAnnihilator) [reply src/modules/RegAuth/Registration/reducer/registrationReducer.ts:1]: > TS1149: File name '/Users/eugenepyatibratov/Desktop/CusDeb/src/modules/RegAuth/Registration/reducer/RegistrationReducer.ts' differs from already included file name '/Users/eugenepyatibratov/Desktop/CusDeb/src/modules/RegAuth/Registration/reducer/registrationReducer.ts' only in casing.

Comment (Gerasimov94) [reply src/modules/RegAuth/Registration/reducer/registrationReducer.ts:1]: done

**Review by CodeAnnihilator — APPROVED** [review]

**Pull request merged**